packages/kbn-profiling-utils/common/profiling.ts

```diff
@@ -220,7 +220,7 @@ function checkIfStringHasParentheses(s: string) {
 }
 
 function getFunctionName(metadata: StackFrameMetadata) {
-  return metadata.FunctionName !== '' && !checkIfStringHasParentheses(metadata.FunctionName)
+  return !checkIfStringHasParentheses(metadata.FunctionName)
     ? `${metadata.FunctionName}()`
     : metadata.FunctionName;
```

Comment:
Can you invert this logic, please? I think it's easy to follow when the truth is handled first:
```suggestion
  return checkIfStringHasParentheses(metadata.FunctionName)
    ? metadata.FunctionName
    : `${metadata.FunctionName}()`;
```